### PR TITLE
Willianpaixao feature/3227 build multi arch docker images

### PR DESCRIPTION
**Pull request by se3000**

### CODE REVIEW

**Review by samsondav — APPROVED** [review]

File: .github/workflows/build-publish-chainlink.yml [hunk 1]
```diff
@@ -35,8 +35,12 @@ jobs:
...
             echo "::set-output name=password::$PASSWORD"
           }
-      - name: Build & Push image
-        uses: docker/build-push-action@v1
+      - name: Docker Setup QEMU
+        uses: docker/setup-qemu-action@v1.0.1
+      - name: Docker Setup Buildx
+        uses: docker/setup-buildx-action@v1.0.2
+      - name: Build and push Docker images
+        uses: docker/build-push-action@v2.1.0
```
Comment (HenryNguyen5) [inline .github/workflows/build-publish-chainlink.yml:43]: build-push-action 2.x removes support for `tag_with_ref` and `tag_with_sha`, see [here](https://github.com/smartcontractkit/chainlink/pull/3650)

**Review by HenryNguyen5 — CHANGES_REQUESTED** [review]

File: .github/workflows/build-publish-chainlink.yml (hunk 1 above)
Comment (willianpaixao) [reply .github/workflows/build-publish-chainlink.yml:43]: Correct, I only checked the changelog for 2.1.x and forgot about the 2.x. These flags are indeed incompatible with 2.x. Need to upgrade the pipeline.
PS: the correct link you wanted to share is https://github.com/docker/build-push-action/blob/master/UPGRADE.md#tags-with-ref-and-git-labels

**Comment by willianpaixao** [comment]
Related #3668

File: .github/workflows/build-publish-chainlink.yml (hunk 1 above)
Comment (se3000) [reply .github/workflows/build-publish-chainlink.yml:43]: Ok, closing for now. @willianpaixao please update your initial PR and then someone will usher the new changes through. Thanks.

Comment (willianpaixao) [reply .github/workflows/build-publish-chainlink.yml:43]: @se3000 @HenryNguyen5 changes done. Please read my comment on the original PR at #3614.

**Pull request closed without merging**